src/main/java/com/laserfiche/repository/api/clients/EntriesClientImpl.java

```diff
@@ -76,8 +77,19 @@ public CompletableFuture<File> exportDocumentWithAuditReason(String repoId, Inte
     }
 
     @Override
-    public CompletableFuture<Void> getDocumentContentType(String repoId, Integer entryId) {
-        return generatedClient.getDocumentContentType(repoId, entryId);
+    public CompletableFuture<GetDocumentContentTypeResult> getDocumentContentType(String repoId, Integer entryId) {
+        return extensionClient.getDocumentContentType(repoId, entryId).thenApply(response -> {
+            String contentType = response.headers().get("content-type");
+            String contentLengthStr = response.headers().get("content-length");
+            if (contentType == null) {
+                throw new RuntimeException("The header doesn't contain 'content-type'.");
```

Comment:
would it be easier for the clients to just return null and let them handle exceptions

Reply:
For this one, if we return null, they don't know why it's null. It could be either content type is null or the content length is null.

Reply:
But that would indeed be easier.

Reply:
Oh, that's a good idea. Let me update.

Reply:
Updated.